bundle_pyinstaller.sh

```diff
@@ -0,0 +1,194 @@
+    pip uninstall -y gnupg && pip install gnupg
+
+    set +x
+    echo "${cc_green}Status: $status done.${cc_normal}"
+}
+
+create_sumo(){
+    echo "${cc_red}No sumo creation available in here.${cc_normal}"
+    exit  # no creation, get sumo from path
+    status="creating sumo tarball"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+
+
+    # source $VENV_DIR/bin/activate
+    # cd $REPOS_ROOT/bitmask_client/
+    # python setup.py sdist --sumo
+}
+
+install_sumo(){
+    if [[ -z $1 ]]; then
+        echo "You need to specify a sumo tarball path parameter."
+        exit 1
+    fi
+
+    status="'installing' sumo tarball"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+
+    source $VENV_DIR/bin/activate
+    SUMO_PATH=$1
+    # from http://stackoverflow.com/a/122340/687989
+    SITE_PACKAGES=`python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"`
+
+    cd $SITE_PACKAGES
+
+    # the file name is like 'leap.bitmask-0.9.0rc1-12-g59bc704-SUMO.tar.gz'
+    # and the root path inside the archive for that file would be
+    # 'leap.bitmask-0.9.0rc1-12-g59bc704-SUMO'
+    base_path=`basename $SUMO_PATH .tar.gz`
+    tar xzf $SUMO_PATH $base_path/src/leap/ --strip-components=2
+}
+
+linux_tweaks(){
+    # - mv bitmask frozen binary to bitmask-app
+    # - cp bitmask-launcher (shell wrapper) to dist/bitmask/bitmask
+    # - cp all the needed helper files, tuf config etc
+    # - rename dist/bitmask distribution folder to bitmask-<release>-<date?>
+    exit
+}
+
+tweak_zope(){
+    # - create bare zope dir in site-packages
+    # - touch zope/__init__.py
+    # - move the zope.interface folder to zope/
+    # - move the zope.proxy folder to zope/
+    # - remove the zope* eggs in site-packages
+    # - remove any refs to these eggs in easy-install.pth ?? (unsure about this, talking from the top of my head) 
+
+    SITE_PACKAGES=`python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"`
+    cd $SITE_PACKAGES
+    mkdir -p zope
+    touch zope/__init__.py
+    # zope/interface and zope/proxy already in place
+    rm -fr zope.{interface,proxy}*.egg-info
+    rm -f zope.{interface,proxy}*-nspkg.pth
+    cd -
+}
+
+run_pyinstaller(){
+    if [[ -z $1 ]]; then
+        echo "You need to specify a bitmask spec file path parameter."
+        exit 1
+    fi
+
+    # make clean_pkg && make pyinst
+    spec_file=$1
+    cd $REPOS_ROOT/bitmask_client
+    make clean
+    pyinstaller $spec_file
+}
+
+if [[ -z $1 || -z $2 ]]; then
+    echo "Bitmask pyinstaller creator."
+    echo "You need to use 2 parameters: bitmask spec path and sumo tarball path."
+    echo "Usage:"
+    echo "  $0 bitmask.spec path/to/sumo.tar.gz"
+    exit 1
+fi
+
+bitmask_spec=`realpath $1`
+sumo_path=`realpath $2`
+
+create_venv
+install_pyinstaller
+clone_repos
+install_requirements
+install_sumo $sumo_path
+tweak_zope
+run_pyinstaller $bitmask_spec
```

Comment:
You mean like we do for `bootstrap_develop.sh:run` -> https://github.com/leapcode/bitmask_client/blob/develop/pkg/scripts/bootstrap_develop.sh#L231 ?
Yes, if we have an use case we could add that.